docs/configuration.html

```diff
@@ -52,100 +52,7 @@
-    </tr>
-    <tr>
-      <td>flush.messages</td>
-      <td>None</td>
-      <td>log.flush.interval.messages</td>
-      <td>This setting allows specifying an interval at which we will force an fsync of data written to the log. For example if this was set to 1 we would fsync after every message; if it were 5 we would fsync after every five messages. In general we recommend you not set this and use replication for durability and allow the operating system's background flush capabilities as it is more efficient. This setting can be overridden on a per-topic basis (see <a href="#topic-config">the per-topic configuration section</a>).</td>
-    </tr>
-    <tr>
-      <td>flush.ms</td>
-      <td>None</td>
-      <td>log.flush.interval.ms</td>
-      <td>This setting allows specifying a time interval at which we will force an fsync of data written to the log. For example if this was set to 1000 we would fsync after 1000 ms had passed. In general we recommend you not set this and use replication for durability and allow the operating system's background flush capabilities as it is more efficient.</td>
-    </tr>
-    <tr>
-      <td>index.interval.bytes</td>
-      <td>4096</td>
-      <td>log.index.interval.bytes</td>
-      <td>This setting controls how frequently Kafka adds an index entry to it's offset index. The default setting ensures that we index a message roughly every 4096 bytes. More indexing allows reads to jump closer to the exact position in the log but makes the index larger. You probably don't need to change this.</td>
-    </tr>
-    <tr>
-      <td>max.message.bytes</td>
-      <td>1,000,000</td>
-      <td>message.max.bytes</td>
-      <td>This is largest message size Kafka will allow to be appended to this topic. Note that if you increase this size you must also increase your consumer's fetch size so they can fetch messages this large.</td>
-    </tr>
-    <tr>
-      <td>min.cleanable.dirty.ratio</td>
-      <td>0.5</td>
-      <td>log.cleaner.min.cleanable.ratio</td>
-      <td>This configuration controls how frequently the log compactor will attempt to clean the log (assuming <a href="#compaction">log compaction</a> is enabled). By default we will avoid cleaning a log where more than 50% of the log has been compacted. This ratio bounds the maximum space wasted in the log by duplicates (at 50% at most 50% of the log could be duplicates). A higher ratio will mean fewer, more efficient cleanings but will mean more wasted space in the log.</td>
-    </tr>
-    <tr>
-      <td>min.insync.replicas</td>
-      <td>1</td>
-      <td>min.insync.replicas</td>
-      <td>When a producer sets acks to "all", min.insync.replicas specifies the minimum number of replicas that must acknowledge a write for the write to be considered successful. If this minimum cannot be met, then the producer will raise an exception (either NotEnoughReplicas or NotEnoughReplicasAfterAppend).
```

Comment:
Actually I just realized that these "hand-written" configurations are more descriptive so we should probably additionally update the `LogConfig` class with these.